src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
@heff has a really good point about accessing methods on a playlist no longer associated with a player. Thinking about it, it does seem that the (playlist)item(), next(), and previous() are state/actions on the *player*, not the playlist. I think I'm convinced we should be doing more of this on the player. Is this a good summary?

```js
player.playlist(); // gets/sets the playlist currently loaded into the player
player.playlistItem(); // gets/sets the index into the currently loaded playlist
player.next(); // moves to the next video in the playlist
player.previous(); // moves to the previous video in the playlist
player.autoadvance(); // gets/sets the autoadvance delay
```